src/features/threads/ShowContractedReplies.js

```diff
@@ -0,0 +1,50 @@
+const Feature = require("../../core/Feature");
+const Threads = require("../../sections/Threads");
+
+class ShowContractedReplies extends Feature {
+  constructor() {
+    super({
+      section: Threads,
+      name: "Show Contracted Replies",
+      default: true,
+      description: "Button to show contracted replies",
+      author: "device",
+      profile: "https://hackforums.net/member.php?action=profile&uid=1255039"
+    });
+  }
+
+  run() {
+    let threadId = 0;
+    const url = "https://hackforums.net/showthread.php?tid=";
+    //  Get thread ID\
+    if ($(".breadcrumb-back-arrow")[1].href.includes("php?fid=105")) {
+      $("input[type=hidden]").each(function() {
+        if ($(this).attr("name") === "tid") {
+          threadId = $(this).attr("value");
+        }
+      });
+      // Creates a "Contracted replies" button
+      let button = document.createElement("button");
+      button.innerHTML = "Contracted Posts";
+      button.className = "button contracted-posts";
+      $(button).css({
+        "padding": "11px",
+        "background-color": "#1f1f1f",
+        "border": "none",
+        "font-size": "14px"
+      });
+      $(button).prependTo($(".float_right.mobile-remove"));
+      button.onclick = function() {
+        location.href = url + threadId + "&mode=contracted";
+      };
+      if (location.href === url + threadId + "&mode=contracted") {
```

Comment:
Since contracted posts are paginated, can you change this if condition to check if it contains contracted mode, like this:
```location.href.includes("&mode=contracted")```